payments.yaml

```diff
@@ -0,0 +1,4 @@
+product_types:
+  'CICACC' : 'CIC report'
+  'LFP' : 'Late filing penalty'
+  'SECURE243' : 'Protect your home from credit reference agencies'
```

Comment:
can you add 'address' to the SECURE243 type?    so... 'home address from credit"

Reply:
was advised to remove the SECURE243 all together. Updated changes below.